code/web/src/modules/user/Profile.js

```diff
@@ -31,16 +35,42 @@ const Profile = (props) => (
     </Grid>
 
     <Grid>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        {/*<Tile style={{}} image={icon}></Tile>*/}
+        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
+      </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-
+        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
+        {/*Description text should be dynamic from user details ie props.user.details.description */}
```

Comment:
It's also worth looking into maybe having a default photo and value for the user bio in the data base that is only changed once they update their information from here. So that we don't have to change the sign up page at all, but rather just work with profile editing specific issues.